testsuite/objects/__init__.py

```diff
@@ -35,3 +35,24 @@ class Authorization(LifecycleObject):
     @abc.abstractmethod
     def add_oidc_identity(self, name, endpoint):
         """Adds OIDC identity provider"""
+
+
+class PreexistingAuthorino(Authorino):
+    """Authorino which is already deployed prior to the testrun"""
+
+    def __init__(self, authorization_url) -> None:
+        super().__init__()
+        self._authorization_url = authorization_url
+
+    def wait_for_ready(self):
+        return True
```

Comment:
I suppose we don't care if the preexisting Authorino instance is NOT ready, right? Tests will fail of course, but that's all right. Any preexisting Authorino instance is supposed to be ready before using this option, I imagine.